packages/core/src/connectors/index.ts

```diff
@@ -92,23 +92,27 @@ const loadConnectors = async () => {
 
 export const getLogtoConnectors = async (): Promise<LogtoConnector[]> => {
   const connectors = await findAllConnectors();
-  const connectorMap = new Map(connectors.map((connector) => [connector.id, connector]));
 
-  const logtoConnectors = await loadConnectors();
+  const virtualConnectors = await loadConnectors();
 
-  return logtoConnectors.map((element) => {
-    const { id } = element.metadata;
-    const connector = connectorMap.get(id);
+  return Promise.all(
+    connectors.map(async (connector) => {
```

Comment:
That is a mapping by sync function? And the `Promise.all` is useless

Reply:
Fixed.